lexer/lexer.go

```diff
@@ -207,6 +208,12 @@ func (l *Lexer) eatCurrentLine() {
 }
 
 func (l *Lexer) eatMultiLineComment() {
+	for {
+		l.eatUntil('*')
+		if char, _ := l.NextChar(); char == '/' {
+			break
```

Comment:
I think we need to handle the case where the comment does not terminate before end of input.